literature.bib

```diff
@@ -6635,21 +6900,19 @@ @article{MH:JSS13
 	year = 2013
 }
 
-@inproceedings{RZS:ICCS09,
-  author = {Andrei Rimsa and Luis Enrique Z{\'{a}}rate and Mark A. J. Song},
-  title = {{Evaluation of Different {BDD} Libraries to Extract Concepts in {FCA} - Perspectives and Limitations}},
-  booktitle = ICCS,
-  location = {Baton Rouge, LA, USA},
-  series = LNCS,
-  volume = {5544},
-  pages = {367--376},
-  publisher = Springer,
-  address = BerlinHeidelberg,
-  doi = {10.1007/978-3-642-01970-8\_36},
-  year = 2009,
-  kc-tags = {BuDDy, CUDD, BDD Evaluation}
```

Comment:
Why is this deleted? Or is this a diff anomaly?

Reply:
I think the entry was placed in the wrong year before so I moved it.